config/meilisearch/offre-de-stage/offre-de-stage.transformation.test.ts

```diff
@@ -1,339 +1,344 @@
+			});
+		});
+
+		context("et que cette dernière est partiellement renseignée", () => {
+			it("retourne les champs renseignés", () => {
+				// Given
+				const localisationAttendue: Meilisearch.OffreDeStage.Localisation = {
+					ville: "Montpellier",
+					departement: "Hérault",
+					codePostal: "34070",
+					region: "Occitanie",
+					pays: undefined,
+					_geo: {
+						lat: undefined,
+						lng: undefined,
+					},
+				};
+
+				const localisation: Strapi.OffreDeStage.Localisation = {
+					adresse: "280 Avenue Germaine Tillion",
+					codePostal: "34070",
+					departement: "Hérault",
+					pays: undefined,
+					region: "Occitanie",
+					ville: "Montpellier",
+					latitude: undefined,
+					longitude: undefined,
+				};
+
+				// When
+				const result = transformerLaLocalisation(localisation);
+
+				// Then
+				assert.deepEqual(result, localisationAttendue);
+			});
+		});
+
+		context("et que cette dernière est complètement renseignée", () => {
+			it("retourne tous les champs", () => {
+				// Given
+				const localisation: Strapi.OffreDeStage.Localisation = {
+					adresse: "280 Avenue Germaine Tillion",
+					codePostal: "34070",
+					departement: "Hérault",
+					pays: "France",
+					region: "Occitanie",
+					ville: "Montpellier",
+					latitude: 3.14159,
+					longitude: -3.14159,
+				};
+
+				const localisationAttendue: Meilisearch.OffreDeStage.Localisation = {
+					ville: "Montpellier",
+					departement: "Hérault",
+					codePostal: "34070",
+					region: "Occitanie",
+					pays: "France",
+					_geo: {
+						lat: 3.14159,
+						lng: -3.14159,
+					},
+				};
+
+				// When
+				const result = transformerLaLocalisation(localisation);
+
+				// Then
+				assert.deepEqual(result, localisationAttendue);
+			});
+		});
+	});
+
+	context("Lorsque je transforme le domaine", () => {
+		context("et que le nom n'est pas défini dans les domaines", () => {
+			it("retourne un tableau vide", () => {
+				// Given
+				const domaines: Array<Strapi.OffreDeStage.Domaine> = [{ nom: undefined }]
+
+
+				// When
+				const actual = transformerLesDomaines(domaines);
+
+				// Then
+				assert.deepEqual(actual, []);
+			});
+		});
+
+		context("et que le nom est défini", () => {
+			it("retourne un tableau avec les domaines", () => {
+				// Given
+				const domaines = [{ nom: Strapi.OffreDeStage.Domaine.Nom.ACHATS }]
+
+
+				// When
+				const actual = transformerLesDomaines(domaines);
+
+				// Then
+				assert.deepEqual(actual, ["achats"]);
+			});
+		});
+
+		context("et que les noms sont parfois définis et parfois non", () => {
+			it("retourne un tableau avec seulement les domaines définis", () => {
+				// Given
+				const domaines = [{ nom: Strapi.OffreDeStage.Domaine.Nom.ACHATS }, { nom: undefined }]
+
+				// When
+				const actual = transformerLesDomaines(domaines);
+
+				// Then
+				assert.deepEqual(actual, ["achats"]);
+			});
+		});
+	});
+
+	context('creerLocalisationFiltree', () => {
```

Comment:
```suggestion
	context('Lorsque je filtre les différents niveaux de localisation', () => {
```
Ou quelque chose de ce genre pour suivre le même schéma ^^